src/c++/perf_analyzer/docs/cli.md

```diff
@@ -284,6 +290,15 @@ collisions.
 
 The default for 'start is `1`, and 'end' is not specified (no bounds).
 
+#### `--serial-sequences`
+
+Enables the serial sequence mode where only 1 request is live per sequence.
+Sequences will be issued in a nearly round robin fashion. Note: It is
+possible that this mode can cause the request_rate_mode to not achieve the
```

Comment:
no underscores. This is documentation, not code.

Reply:
thank you. removed